search/search_repository_whitebox_test.go

```diff
@@ -526,6 +526,7 @@ func TestIsOperator(t *testing.T) {
 	testData := map[string]bool{
 		AND:   true,
 		OR:    true,
+		OPTS:  true,
```

Comment:
I know this is not part of your task here. But can you please also add all the other constants we have defined and give them an expected `false` result?

Reply:
Done. 8a204ed5d216a4603c8a39e879d7a49be4bfcc81

Reply:
I've been thinking about this and I came to the conclusion that `$OPTS` isn't really an operator at all. It is just special but we shouldn't treat it as operator. Sorry for not seeing this at first.

Reply:
I have removed it from `isOperator` function. 04aa0e30c312e0f21ce1894185999e69f51beaa7